libs/camera/frustum_intersection_graph.cpp

```
@@ -259,21 +259,16 @@ void FrustumIntersectionGraph::set_manual_mode(const std::vector<CameraGraphNode
-                                                   const std::vector<CameraId>& depth_ids,
-                                                   bool allow_stereo_track_for_depth,
-                                                   const MulticamManualSetup& manual_setup)
-    : depth_ids_(depth_ids) {
-  TraceErrorIf((mode == MulticameraMode::Manual) == manual_setup.empty(),
+FrustumIntersectionGraph::FrustumIntersectionGraph(const camera::Rig& rig, const FigSettings& settings)
+    : FrustumIntersectionGraph(BuildCameraGraph(rig), settings) {}
+
+FrustumIntersectionGraph::FrustumIntersectionGraph(const std::vector<CameraGraphNode>& graph,
+                                                   const FigSettings& settings)
+    : depth_ids_(settings.depth_ids) {
+  TraceErrorIf((settings.mode == MulticameraMode::Manual) == settings.manual_setup.empty(),
```

Clang (14.0.6)

[warning] do not call c-style vararg functions

(cppcoreguidelines-pro-type-vararg)